### PR TITLE
Search field test ml

### PR DESCRIPTION
**Pull request by gercen8905**
Guys, it's version without deep refactoring

### CODE REVIEW

**Review by czajkaaga — APPROVED** [review]
it looks nice; simply and clean code :)

**Comment by dominikbodzioch** [comment]
Nice use of test annotations! :)

Something happened to the path: src/test/java/**com/epam/qalab**/com/epam/qalab/SearchFieldTest.java
When I checkout to your branch, I can't run test, because it doesn't match with package declaration in file.

Even when I correct package and java version in pom, I still can't run the test.
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
'pom' probably needs more dependencies (I added them in my pull request).

After I complete the missing dependencies, I still have a problem with chromedriver:
"java.lang.IllegalStateException: The driver executable must exist: D:\2022\02_25_amazon_selenium_group1\src\test\resources\chromedriver"
(chromedriver.exe is present under this path)

Unfortunately, I can't run the tests and say anything more about the logic.
I am not sure, but in "tearDown()" method you probably should also call "close" method on the driver (before "quit").

**Comment by czajkaaga** [comment]
> After I complete the missing dependencies, I still have a problem with chromedriver: "java.lang.IllegalStateException: The driver executable must exist: D:\2022\02_25_amazon_selenium_group1\src\test\resources\chromedriver" (chromedriver.exe is present under this path)

@dominikbodzioch in my case I used ".exe" at the end of path to my driver file and it works. You can try this:
**\src\test\resources\chromedriver.exe**

**Comment by gercen8905** [comment]
@dominikbodzioch review changes plz. Suppose that now tests should run.

**Comment by dominikbodzioch** [comment]
> @dominikbodzioch review changes plz. Suppose that now tests should run.

"src/test/resources/chromedriver.**exe**" - only this is required to run and pass tests (.exe)

**Comment by gercen8905** [comment]
@dominikbodzioch I think it's because of different operational systems (I have MacOS). When I append .exe to my path test are not executed

**Review by dominikbodzioch — APPROVED** [review]
In a real project, we should adapt to the target environment (driver extension). 
In our conditions, let this variable remain in the "comfortable for the author" version.

**Comment by gercen8905** [comment]
In a real project I think DevOps will run our tests in Jenkins and I heard that there could be some issues with different operation systems (

**Comment by kristinapeneva** [comment]
Hello,
The tests run smoothly and are easy to read. I like that you defined correct and incorrect variables on the top and used a separate method search().
In my opinion you may only replace \" in xPath with ', which is shorter. Anyway, very good job!

**Review by kristinapeneva — COMMENTED** [review]
I agree with the changes

File: src/test/java/com/epam/qalab/SearchFieldTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.List;
+
+
+public class SearchFieldTest {
+    WebDriver driver;
```
Comment (juliaHrabovska) [inline src/test/java/com/epam/qalab/SearchFieldTest.java:24]: All variables can be private

**Pull request merged**